lib/modules/address.js

```diff
@@ -56,6 +59,74 @@ AddressModule.prototype.getPublishEvents = function() {
   ];
 };
 
+/**
+ * Will process each output of a transaction from the daemon "tx" event, and contruct
+ * an object with the data for the message to be relayed to any subscribers for an address.
+ *
+ * @param {Object} messages - An object to collect messages
+ * @param {Transaction} tx - Instance of the transaction
+ * @param {Output} output - The output being handled
```

Comment:
I think the method signature changed since the JS Doc was constructed. There seems to be a mismatch in the number of options?